fdbserver/LogSystemPeekCursor.actor.cpp

```diff
@@ -895,8 +934,10 @@ ILogSystem::SetPeekCursor::SetPeekCursor(std::vector<Reference<LogSet>> const& l
 	for (int i = 0; i < logSets.size(); i++) {
 		for (int j = 0; j < logSets[i]->logServers.size(); j++) {
 			bool returnEmptyIfStopped =
-			    ((SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST && end != std::numeric_limits<Version>::max())
-			         ? (bestSet < 0 || bestServer < 0 || (i == bestSet && j == bestServer))
+			    ((SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST && bestSet >= 0 &&
+			      end != std::numeric_limits<Version>::max())
```

Comment:
already check `end != std::numeric_limits<Version>::max()` in the helper, this is redundant